content/en/docs/guides/performance.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Performance Best Practices
```

Comment:
This formats a little weirdly.  I think it might be better to just say:

```
# Performance Best Practices
```

Reply:
What part of the formatting is weird? I followed the other guides formatting and using `title` is how the guide appears in the navigation menu. I don't really see anything strange when I serve it locally?

Reply:
When I did "View file" it showed up as a sort of table.  If that's expected for the structure on grpc.io, then it's fine.

Reply:
Oh yeah that's not how it looks in the real display. I think there's some formatting logic omitted from the rendering here.